RecoJets/JetProducers/interface/hltMVAJetPuId.h

```diff
@@ -0,0 +1,82 @@
+#ifndef RecoJets_JetProducers_plugins_hltMVAJetPuId_h
+#define RecoJets_JetProducers_plugins_hltMVAJetPuId_h
+ 
+#include "DataFormats/JetReco/interface/Jet.h"
+#include "DataFormats/JetReco/interface/PFJetCollection.h"
+#include "DataFormats/VertexReco/interface/VertexFwd.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+ 
+#include "TMVA/Tools.h"
+#include "TMVA/Reader.h"
+#include "TMVA/Tools.h"
+#include "TMVA/Reader.h"
+ 
+#include "DataFormats/JetReco/interface/PileupJetIdentifier.h"
+
+
+class hltMVAJetPuId {
```

Comment:
sorry, why was the class name changed to hltMVAJetPuId?
The class should stay with MVAJetPuId.

The comment I made earlier was only about the module instance name in hltMVAJetPuId_cff.py (thank you for changing that one)